test/cmdlineTests/~documentation_examples/test.sh

```diff
@@ -39,6 +39,10 @@ do
     then
         opts+=(--ignore-warnings)
     fi
+    if grep "// This will only compile with via-ir" "$f" >/dev/null
```

Comment:
```suggestion
    if grep "// This will only compile via IR" "$f" >/dev/null
```

By the way, `via-ir` seems to be a new variant of spelling this. Maybe it would be better to stick to `via IR` or `via-IR` that we already use? It's a small thing, but it looks sloppy when it's spelled differently each time and this variant is new so we still have a chance to avoid introducing it :)

Reply:
There's one instance of this form left in `control-structures.rst`.

Also, in the message above it should really be `compile via IR` as I suggested. We *compile Solidity to bytecode by going through (via) the IR pipeline (IR code generator)*. I guess there are situations where using it as an adverb does not work so somehow we ended up with this horrible `via-IR` thing used as a noun, but something like `compile with via-IR` is one of the exact situations where it does work :) Does it really not bother you as weird?